[REEF-1947] xUnit error in TestHelloREEFClient

jwang98052 opened the pull request:
The test is for yarn and called from a test console. So remove [Fact] for now.

JIRA: [REEF-1947](https://issues.apache.org/jira/browse/REEF-1947)
This closes #

tcNickolas commented:
Another thing about `TestHelloREEFClient` - it failed in a [recent AppVeyor build](https://ci.appveyor.com/project/ApacheSoftwareFoundation/reef/build/2021-master/job/pb5dv3q0acn1g6xe/tests) (and in [the build](https://ci.appveyor.com/project/ApacheSoftwareFoundation/reef/build/2032-master/job/2aafs4ivnxs5af22/tests) for this pull request as well) because xUnit tries to execute it in parallel with other functional tests and they compete for the jar file. It has to have `[Collection("FunctionalTests")]` (same as all other tests which inherit from `ReefFunctionalTest`) so that xUnit knows to run them all sequentially, not in parallel.

jwang98052 commented:
@tcNickolas Thanks for the comment! I have updated it. 
@motus This test can be called from a console test driver with command line arguments. For now, I changed it to hard code some test parameters.
